src/layout/Footer.tsx

```diff
@@ -30,9 +31,18 @@ export const Footer = () => {
         bgcolor: 'info.light',
         display: 'grid',
         gridTemplateColumns: { xs: '1fr', md: '1fr 1fr 1fr' },
-        gap: '3rem',
+        gridTemplateRows: { md: 'auto 1fr' },
+        columnGap: '3rem',
         color: 'primary.main',
       }}>
+      <Button
+        sx={{ width: 'fit-content', justifySelf: 'center', gridColumn: { md: '1 / span 3' } }}
+        data-testid={dataTestId.footer.toTopButton}
+        onClick={() => window.scroll({ top: 0, behavior: 'smooth' })}
+        startIcon={<KeyboardArrowUpIcon />}
+        aria-label={t('footer.to_top')}>
```

Comment:
Burde ikke være behov for denne når innholdet er helt likt 🤔 
```suggestion
        startIcon={<KeyboardArrowUpIcon />}>
```